HomebrewFormula

```diff
@@ -0,0 +1 @@
+pkg/brew
```

Comment:
Symlink to allow for using custom folder as the brew formula location. Hat tip to ... ripgrep for this pattern?